genesis/vaulta_chain_id.txt

```diff
@@ -0,0 +1,3 @@
+{
+ "chain_id": "aca376f206b8fc25a6ed44dbdc66547c36c6c33e3a119ffbeaef943642f0e906"
+}
```

Comment:
Since these files have json in them it probably makes sense for the files to be .json instead of .txt?

Reply:
Sure, I'll update them. 

Personally I think it would be nicer to have 3 subdirectories: 
- `vaulta_mainnet`
-  `jungle_testnet`
- `kylin_testnet`

What do you guys think?

Reply:
> Since these files have json in them it probably makes sense for the files to be .json instead of .txt?

https://github.com/AntelopeIO/spring/pull/1374/commits/277b08cb13e438a85f703506bb995b8da9188f1a